schemas/zcrypto.py

```diff
@@ -76,6 +79,105 @@
     "asn1_oid":String(),
 })
 
+# /go/src/net/ip.go: type IPNet struct
+ip_net = SubRecord({
+    "IP": Binary(),
+    "Mask": Binary(),
+})
+
+# x509/x509.go: type GeneralSubtreeIP struct
+general_subtree_ip = SubRecord({
+    "Data": ip_net,
+    "Max": Integer(),
+    "Min": Integer(),
```

Comment:
Shit, are these actually leading capital letter in ZCrypto?

Reply:
I think so -- they have no `json` tag, and I don't see a custom Marshal function (https://github.com/zmap/zcrypto/blob/master/x509/x509.go#L663).